### PR TITLE
Suggestion_on_religious_inclusivity

### PR DESCRIPTION
**Pull request by ERSlayton**
I would change the word g-d to a different village name, as religious Jewish practice does not allow for the writing of the word g-d. Although I anticipate this being an issue for a minority of people, it is something to consider to be more inclusive.

Please delete this line and the text below before submitting your contribution.

---

Thanks for contributing! If this contribution is for instructor training, please send an email to checkout@carpentries.org with a link to this contribution so we can record your progress. You’ve completed your contribution step for instructor checkout just by submitting this contribution.  

Please keep in mind that lesson maintainers are volunteers and it may be some time before they can respond to your contribution. Although not all contributions can be incorporated into the lesson materials, we appreciate your time and effort to improve the curriculum.  If you have any questions about the lesson maintenance process or would like to volunteer your time as a contribution reviewer, please contact The Carpentries Team at team@carpentries.org.  

---

### CODE REVIEW

**Comment by angela-li** [comment]
Thank you for resubmitting this! We will likely need to replace it with the name of a different village (so it's possible to see the output in the lesson text), so I will bring this to my co-maintainers and see how we'd like to fix this. This is much appreciated and brings to light a perspective on this lesson we hadn't previously considered - thank you!

**Comment by ErinBecker** [comment]
Thanks for this comment @ERSlayton. I've been mulling over this issue since I saw it come in yesterday. I am Jewish, and do have a pretty solid background in halacha, so I know the issue you're referring to. Religious inclusivity is very important to me, and I don't want to be seen as dismissive of your comment, which is why I mention that my response is grounded in a religious Jewish background. 

According to most Jewish legal sources, the written name of God, when written in any language other than Hebrew, does not have kedusha (i.e. is not holy and therefore doesn't require any special treatment). Aish.com has [a good article on this](https://www.aish.com/atr/Writing_the_Name_God.html). I think this argument is particularly strengthened by the fact that this village name isn't actually intended to refer to "God" as in a divine being, but just happens to be the letters g-o-d. As the data used in this workshop is from a real study, and refers to an actual place (which maps to the latitude and longitude data also used in the workshop), I would be very hesitant to change the name arbitrarily. 

Maybe this issue could be addressed by adding a map showing the location of the villages? Or looking to see if there is an appropriate diacritic (e.g. accent mark) that might be added to the village name to distinguish it visually? I've certainly experienced (and seen others experience) some discomfort or hesitation in saying this village name out loud during workshops and have pronounced it with a hard "O" (G-oh-d) to differentiate it. 

In short, I would like to explore other options of making it clear to Instructors and Learners that this string of letters refers to an actual place name. What do you all think?

**Comment by ERSlayton** [comment]
Hello Erin,

Thank you for your detailed and personal response.

I was just trying to bring this to your attention, which I now see was was necessary as it is clear how much thought you have put into this issue already.

Your completely right that the variables within the data set should not be changed. More my suggestion, if you wanted to take action on this, would be to use the name of another city within the data set for the practical aspects of the workshop. I have done so for the version I teach (I use it frequently as I have found this lesson to be so helpful! It really is a great lesson).

This may require less direct engagement with the word that might cause some people discomfort. However, you have a strong grasp of what I was referring to in my comment, and I defer to your judgement.

Thank you for taking such a thoughtful and serious look at my suggestion.

All the best,

Emma Slayton
________________________________

**Comment by ErinBecker** [comment]
Wow, thank you so much for that message. It really means a lot.

Could you please share which of the other village names you use in your workshops? I remember exploring other options and not having them work as well for the visualizations, but would love to test out whichever combination you're using and see if we can apply that to the official lesson.

**Comment by ERSlayton** [comment]
Hello Erin,

I have been using Chirodzo, though I don’t know how that works better or worse with the visualization from the other cities.

All the best,

Emma


--

Emma Slayton, PhD
Data Curation, Visualization, and GIS Specialist
Carnegie Mellon University Libraries
4408 Wean Hall
Pittsburgh, PA 15213
eslayton@andrew.cmu.edu

**Comment by ErinBecker** [comment]
Thanks @ERSlayton! That's a great suggestion. @angela-li - could you please follow-up to see if that change would work for the visualisations? I'm not sure how much data we have for each of the villages and if it would work for the examples we currently have. If we can "plug and play" by just changing the village name and keeping all the code, that would be ideal. If we need to change examples, that would be a bit more involved and in that case I would probably prefer keeping the current village and adding a map or pronounciation note.

**Comment by kerchner** [comment]
@ErinBecker @ERSlayton great conversation thread here :)  I taught this (really good!) lesson, and it never crossed my mind, probably because - as noted in the link above - the context here matters, as far as what the term is referencing and what it's not.

**Comment by juanfung** [comment]
Resolved #249
